testing/trino-testing/src/main/java/io/trino/testing/BaseOrcWithBloomFiltersTest.java

```diff
@@ -90,7 +70,7 @@ private void assertBloomFilterBasedRowGroupPruning(@Language("SQL") String sql)
                 results -> assertThat(results.getRowCount()).isEqualTo(0));
 
         assertQueryStats(
-                getSession(),
+                isSkipRedundantSort(getSession()),
```

Comment:
as in https://github.com/trinodb/trino/pull/11732#discussion_r866835025 :

> `skip_redundant_sort` is a compatibility flag, it shouldn't be relied on for new futures (or tests of new features).

please remove its usage from here